apps/obsidian/src/components/TldrawViewComponent.tsx

```diff
@@ -36,6 +42,15 @@ export const TldrawPreviewComponent = ({
   const saveTimeoutRef = useRef<NodeJS.Timeout>();
   const lastSavedDataRef = useRef<string>("");
 
+  const customShapeUtils = [
+    ...defaultShapeUtils,
+    DiscourseNodeUtil.configure({
```

Comment:
Interesting choice to use `.configure` here.  As I understand it, `.configure` is used to pass optional configuration that alters the default behavior of a shape.  But here it is being used to pass required context for every instance of the shape.  Another way would be to use a factory function to create the util.

Reply:
We could think outside of ShapeUtils for these types of required parameters.